bakery/src/tests/test_cli.js

```diff
@@ -26,82 +106,108 @@ test('stable flow in content distribution pipeline', async t => {
   } catch {}
   await fs.copy(`${dataDir}/${bookId}`, `${outputDir}/${bookId}`)
 
+  const commonArgs = [
+    '--cops=..',
+    `--data=${outputDir}`,
+    '--persist'
+  ]
+
   // Build local cops-bakery-scripts image
   const scriptsImageBuild = spawn('docker', [
     'build',
     'src/scripts',
     '--tag=localhost.localdomain:5000/openstax/cops-bakery-scripts:test'
-  ], { stdio: 'inherit' })
+  ])
   await completion(scriptsImageBuild)
 
   const assemble = spawn('node', [
     'src/cli/execute.js',
-    '--cops=..',
-    `--data=${outputDir}`,
-    '--persist',
+    ...commonArgs,
     'assemble',
     bookId
-  ], { stdio: 'inherit' })
-  await completion(assemble)
-  t.truthy(fs.existsSync(`${outputDir}/${bookId}/assembled-book/${bookId}/collection.assembled.xhtml`))
+  ])
+  const assembleResult = await completion(assemble)
+  t.truthy(fs.existsSync(`${outputDir}/${bookId}/assembled-book/${bookId}/collection.assembled.xhtml`), formatSubprocessOutput(assembleResult))
 
   const assembleMeta = spawn('node', [
```

Comment:
Minor: I believe this task is also only for distribution, so maybe it can be used to fork the dist branch instead of `bakeMeta` if you're trying to keep everything encapsulated in `branchDistribution`?  I may be forgetting some interdependencies there, though, and this is a bit more logical in terms of ordering flow, so ... :shrug:

Reply:
I decided to branch them where they ceased to have any common steps, in this case, both still need to bake, so I saw no reason to branch here. It was more about taking advantage of already completed steps than logical separation to keep CI times dow (clearly, as I used one test to cover both pipelines)